pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -304,7 +307,7 @@ func manageKubeSchedulerConfigMap_v311_00_to_latest(lister corev1listers.ConfigM
 		}
 	}
 
-	requiredConfigMap, _, err := resourcemerge.MergeConfigMap(configMap, "config.yaml", nil, kubeSchedulerConfiguration, operatorSpec.ObservedConfig.Raw, operatorSpec.UnsupportedConfigOverrides.Raw)
+	requiredConfigMap, _, err := resourcemerge.MergeConfigMap(configMap, "config.yaml", nil, kubeSchedulerConfiguration, operatorSpec.UnsupportedConfigOverrides.Raw)
```

Comment:
@damemi I am removing `operatorSpec.ObservedConfig.Raw` from the merging op. It's quite dangerous to have it here as depending on the observers, invalid data can be written into `kubescheduler.config.k8s.io/v1beta1`. The same holds for `operatorSpec.UnsupportedConfigOverrides.Raw`. I guess it was used to pass location of the policy file into the config. Though, it's no longer a case. Also, I am not aware of any other observed bits suitable for `kubescheduler.config.k8s.io/v1beta1`.

Reply:
Sounds good to me, yeah that was used to merge in the policy file name before and is no longer used.